packages/ckeditor5-build-classic/webpack.config.js

```diff
@@ -72,6 +72,15 @@ module.exports = {
 							injectType: 'singletonStyleTag',
 							attributes: {
 								'data-cke': true
+							},
+							insert: function addToStyles( styleElement ) {
+								/* eslint-disable no-undef */ // TODO: figure out why window & document are undefined
```

Comment:
add your username to the todo: `TODO (ywsang):` so people know who to ask about this :)